examples/child_window.rs

```diff
@@ -56,15 +56,21 @@ fn main() -> Result<(), impl std::error::Error> {
                     event: KeyEvent { state: ElementState::Pressed, .. },
                     ..
                 } => {
+                    let child_index = self.windows.len() - 1;
                     let parent_window = self.windows.get(&self.parent_window_id.unwrap()).unwrap();
-                    let child_window = spawn_child_window(parent_window.as_ref(), event_loop);
+                    let child_window =
+                        spawn_child_window(parent_window.as_ref(), event_loop, child_index as i32);
                     let child_id = child_window.id();
                     println!("Child window created with id: {child_id:?}");
                     self.windows.insert(child_id, child_window);
                 },
                 WindowEvent::RedrawRequested => {
                     if let Some(window) = self.windows.get(&window_id) {
-                        fill::fill_window(window.as_ref());
+                        if window_id == self.parent_window_id.unwrap() {
+                            fill::fill_window(window.as_ref(), 0xff181818);
+                        } else {
+                            fill::fill_window(window.as_ref(), 0xffbbbbbb);
```

Comment:
Maybe we should just cycle color for child?

Reply:
Leaving the parent window empty like in the window example and then just the childs are filled?

Reply:
The parent will have the default color, and childs will have different color for each of them.

Reply:
You want the childs colors to be deterministic or random? Mostly asking because it changes wherever I make the color a function of their position or their window id

Reply:
Similar how you did with the position, but also with color, so it'll be _deterministic_ defined by the number of child you've opened.

Reply:
The index I'm using to calculate the position right now is a throwaway value because the child object doesn't already have a way for me to store it, that's why I said making the color a function of the position

Reply:
I mean, how you exactly implement it doesn't really matter. I'd probably just make a counter for number of child windows opened and use it to generate things.

Reply:
I think I may be misunderstanding something, doesn't redraw_request get called for each frame? I thought using a counter instead of a value linked to the child window would end up with every child having the same color because they'd end up being filled with the same color as whatever the last counter number dictated

Reply:
counter is tied to a child window, so it stays the same for the window your drawing. it also means that you store that color somewhere.